helm/templates/configmap-bootstrap.yaml

```diff
@@ -0,0 +1,166 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: db-bootstrap
+  labels:
+    {{- include "gateway.labels" . | nindent 4 }}
+data:
+  gateway-ha-persistence-postgres.sql: |
```

Comment:
hmm is it necessary to have sql as configmap? How about copying during dockerfile as it does not change that often

Reply:
well, it depends.
in that case user needs a customised postgres/mysql image.
or, using these images, for instance, in initContainer first install curl + get sql scripts and then run psql/mysql commands.

Reply:
I see. How about adding such explanation to readme? so that user can customize to fit their needs

Reply:
README updated